Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -120,6 +174,82 @@ public static string[] GetLanguages()
             return localizationTable.Keys.ToArray();
         }
 
+        public static void LoadConfigFile(string pathToConfigFile)
+        {
+            localizationConfigurations = new Dictionary<string, LanguageData>();
+
+            if (File.Exists(pathToConfigFile) == false)
+            {
+                Debug.ULogErrorChannel("LocalizationTable", "No config file found at: " + pathToConfigFile);
```

Comment:
actually the game is a bit too reliant on config.xml, I didn't realize that not having it causes tons of null key errors. Ill relook how the system works when the file is missing. It should probably be an error, but the game should at least still show english or at least localization strings when its missing.